src/hotspot/share/runtime/arguments.cpp

```diff
@@ -1116,7 +1116,7 @@ void Arguments::print_on(outputStream* st) {
     if (len == 0) {
       st->print_raw_cr("<not set>");
     } else {
-      st->print_raw_cr(path, len);
+      st->print_raw_cr(path, (int)len);
```

Comment:
This seems like a reasonable fix.

As a potential follow-up change: I can't help but notice that outputStream::print_raw_cr just delegates to outputStream::write, which takes a size_t argument, so maybe we should align the types?

Reply:
Or change the second parameter of print_raw() and print_raw_cr() to size_t?